test/DataMCSpectraComparison/histosSimplified.py

```diff
@@ -0,0 +1,533 @@
+#!/usr/bin/env python
+
+
+# Set to True in order to select e-mu and dielectron final states
+Electrons = False
+# Flag, setting whether we work with simulation or data
```

Comment:
As you now included a mechanism to decide isMC based on the file thing, please add a comment that this should NOT be changed manually without a good reason, anymore.